packages/mikado_reborn/src/components/List/List.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="mkr__list" :class="[bgItemVariantClass, listSizeClass]">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+export const BgItemVariant = {
+  odd: 'odd',
+  even: 'even',
+  hover: 'hover',
+};
+
+export const ListSize = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+};
+
+@Component
+export default class List extends Vue {
+  @Prop({
+    type: String,
+    default: undefined,
+    validator: (variant) => Object.values(BgItemVariant).includes(variant) || variant === undefined,
+  })
+  bgItemVariant?: keyof typeof BgItemVariant
+
+  @Prop({
+    type: String,
+    default: ListSize.medium,
+    validator: (variant) => Object.values(ListSize).includes(variant),
+  })
+  size!: keyof typeof ListSize
+
+  get bgItemVariantClass() {
+    return this.bgItemVariant ? `--${this.bgItemVariant}` : undefined;
+  }
+
+  get listSizeClass() {
+    return `--${this.size}`;
+  }
```

Comment:
Please format :
`[block]__[element]--[modifier]`
Would be `mkr__list--${this.bgItemVariant}` here.

What about directly using the class attr for such straightforward logic ? Can be all clear at once which classes will be applied.